lab-ron/test/test2-greetings.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var main = require(__dirname + '/../index');
+
+
+describe('testing module index', function(){
+  describe('testing main function', function(){
+    it('process.argv[2] = should return "hello slug"', function(){
+      // assert.equal(main(), 'hello slug');
+      process.argv=['node', 'index.js', 'slug'];
+    });
+  });
+});
```

Comment:
ESLint error is popping up here. 

Also, this test is not testing main() - in fact nothing is being testing. Tt will always pass.  Can you see why?